### PR TITLE
[DO NOT MERGE] Testing remarks without endline separation.

### PR DESCRIPTION
**Pull request by carlossanlop**
Investigation to support this PR: https://github.com/dotnet/runtime/pull/46716

If we backport the comments to triple slash saving lines of code by removing extra empty lines, would the separation would look fine in MS Docs once those comments are sent back to dotnet-api-docs?

### CODE REVIEW

**Comment by carlossanlop** [comment]
@maryamariyan this is just a testing PR but the bot was unable to apply a label. Do you know if the bot reads the list of files in the PR to decide? I was expecting it to add the IO.Compression label.

**Comment by maryamariyan** [comment]
@carlossanlop, if you remember we did not have enough issues labeled so the labeler today only reads the title and description.

Only one issue is labeled as area-System.IO.Compression, (list [here](https://github.com/dotnet/dotnet-api-docs/labels/area-System.IO.Compression)), and that [one]((https://github.com/dotnet/dotnet-api-docs/issues/834)) is actually an issue not a PR. 

So there would have been no way for it would have predicted for this one right off the bat.

File: xml/System.IO.Compression/BrotliEncoder.xml [hunk 1]
```diff
@@ -60,9 +60,7 @@
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentOutOfRangeException">
           <paramref name="quality" /> is not between the minimum value of 0 and the maximum value of 11.
-
```
Comment (carlossanlop) [inline xml/System.IO.Compression/BrotliEncoder.xml:63]: Removing the endlines that surround an -or-` causes MS Docs to show everything into one line. The tool needs to preserve empty lines.

https://review.docs.microsoft.com/en-us/dotnet/api/system.io.compression.brotliencoder.-ctor?view=net-5.0&branch=pr-en-us-5213

image

File: xml/System.IO.Compression/BrotliEncoder.xml [hunk 2]
```diff
@@ -201,11 +199,11 @@
         <returns>A number representing the maximum compressed length for the provided input size.</returns>
         <remarks>
           <format type="text/markdown"><![CDATA[
-
 ## Remarks
-
-Returns 1 if `inputSize` is 0.
-
+How would it look in the MS Docs Preview.
```
Comment (carlossanlop) [inline xml/System.IO.Compression/BrotliEncoder.xml:203]: Reconfirmed. Empty lines need to be preserved.
https://review.docs.microsoft.com/en-us/dotnet/api/system.io.compression.brotliencoder.getmaxcompressedlength?view=net-5.0&branch=pr-en-us-5213
image

Comment (carlossanlop) [reply xml/System.IO.Compression/BrotliEncoder.xml:203]: @safern @gewarren when the time comes where we have DLLs generated by the runtime build alongside their fully documented intellisense XMLs, we should test feeding them to mdoc and see if it is smart enough to determine that _one_ single endline should be rendered as a full `<br />` in MS Docs.

**Pull request closed without merging**